install-apps.sh

```diff
@@ -0,0 +1,3 @@
+#!bin/bash
```

Comment:
You should use an absolute path to the interpreter.
That is
```suggestion
#! /bin/bash
```